src/agent0/core/hyperdrive/interactive/local_hyperdrive_agent.py

```diff
@@ -534,6 +535,8 @@ def get_positions(
             Whether to show positions closed positions (i.e., positions with zero balance). Defaults to False.
             When False, will only return currently open positions. Useful for gathering currently open positions.
             When True, will also return any closed positions. Useful for calculating overall pnl of all positions.
+        calc_pnl: bool
+            Whether to return the pnl for the current position. Only used if the chain config's `calc_pnl` is False.
```

Comment:
I've lost track of the flow here but I think you'd want a warning message here as well? Maybe just here and not the other ones?